host_reference/README.md

```diff
@@ -0,0 +1,32 @@
+# QUDA 1.0.0
+
+## host_reference
+
+This directory contains host side reference code that cross checks the GPU kerels in QUDA
+with host side calculated routines. Currenly supported reference routines for matrix 
+operators are:
+
+	 1. Wilson dslash
+	 2. Clover dslash
+	 3. Twisted mass
+	 4. Twisted clover
+	 5. Non-degenerate twisted mass/clover	
+	 6. Staggered dslash
```

Comment:
I assume this is both staggered and HISQ, please update!

Reply:
fixed